### DISCUSSION TITLE
Can't Use Default Spacing Options

### DISCUSSION THREAD

**Discussion by ibrahim-kaya** [tailwindlabs/tailwindcss#5071] [Help] [answered]
Hi,

I've just started a new project in Laravel. But I'm facing a weird problem with Tailwind. I can't use all margin or padding spacing opitons like m-5 or p-4 or mb-4 and so on... Weird thing is I can use mt-5 but can't use m-5. How can I fix this issue?

Screenshot: https://prnt.sc/1f5n605

**Comment by simonswiss** [comment]
Hey!

This definitely sounds like some issue with the way your tooling processes PostCSS.

Be sure to go through the [Troubleshooting section of the JIT docs](https://tailwindcss.com/docs/just-in-time-mode#troubleshooting).

You might need to use the Tailwind CLI to compile Tailwind, or add the `TAILWIND_MODE=watch` flag in your dev server script.

**Reply by ibrahim-kaya** [reply]
Thanks. I checked the link that you send me. Searched a while, and realized the JIT (Just In Time) feature was on. Thank you!

**Comment by ibrahim-kaya** [comment, marked as answer]
I figured it out. The new JIT (Just In Time) feature was enabled by default. I removed `mode: 'jit',` line in **tailwind.config.js** and compiled again. And problem solved. Thanks!
